snmp/datadog_checks/snmp/data/default_profiles/ibm-datapower-gateway.yaml

```diff
@@ -227,7 +222,7 @@ metrics:
           1: other
           6: ethernet
           131: tunnel
-          135: vlan
+          135: v_lan
```

Comment:
Shall we fix this for consistency sake? or leave it as it? the original text is:

```
vLAN(135)
```

Reply:
Leaving it as it since it's common to use `vlan` term